eitprocessing/datahandling/continuousdata.py

```diff
@@ -87,6 +89,27 @@ def copy(
         obj.unlock()
         return obj
 
+    def __add__(self: T, other: T) -> T:
+        return self.concatenate(self, other)
+
+    def concatenate(self: T, other: T, newlabel: str | None = None) -> T:  # noqa: D102, will be removed soon
```

Comment:
`ContinuousData` now has two `concatenate()` methods. This one is missing the `derived_from` attribute.

Reply:
I realized this. I think it probably happened at some stage during rebasing where both of us had added it at separate parts of the module so it didn't create a conflict. I hadn't gotten to the point of figuring out which one was correct or if both are partially correct, so hadn't resolved it yet.